ERROR    services.travel_service:travel_service.py:70 Ошибка при завершении путешествия 1